native/cocos/bindings/jswrapper/v8/Object.cpp

```diff
@@ -401,13 +401,23 @@ Object *Object::createJSONObject(const ccstd::string &jsonStr) {
 }
 
 Object *Object::createJSONObject(std::u16string &&jsonStr) {
-    auto v8Str = v8::String::NewExternalTwoByte(__isolate, ccnew internal::ExternalStringResource(std::move(jsonStr)));
+    auto *external = ccnew internal::ExternalStringResource(std::move(jsonStr));
+    external->addRef();
```

Comment:
The default reference count is 1, should add ref here?

Reply:
I think about again. It seems doesn't need the reference count here.
Because `auto v8Str = v8::String::NewExternalTwoByte(__isolate, external);` will keep the external buffer until GC comes, so it's safe to invoke `freeMemory` after `v8::JSON::Parse`.

I will update it.

Reply:
UPDATED, remove reference count.